src/jit/gentree.cpp

```diff
@@ -13415,4 +13415,42 @@ bool GenTree::isCommutativeSIMDIntrinsic()
         return false;
     }
 }
+#endif //FEATURE_SIMD
+
+
+// Initialize Multi-reg Return Type Descriptor given type handle.
```

Comment:
//------------------------------------------------------------------------
// <Function name>: <Short description of the function>
//